platform/osgi/platform-osgi-configadmin/src/main/java/org/codice/felix/cm/file/WrappedPersistenceManager.java

```diff
@@ -36,6 +36,10 @@
     this.persistenceManager = persistenceManager;
   }
 
+  protected PersistenceManager getInnerPersistenceManager() {
```

Comment:
❗️ Can be private now or simply removed since the calling code is a method of this class now.

Reply:
This is not entirely correct unless you assumed we would make the **field** it returns protected or package private? This method is called in a child class. We still need protected or package-private access on this method.

Reply:
Sorry, I thought it was only called in this class.